### DISCUSSION TITLE
Ouch is all I can say

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#500] [Galaxy Trivia] [answered]
🦷 + 👑

**Comment by ParanoidUser** [comment]
@rizkygunardi, @ryuuzera, @iamharshil, @demariamed any ideas? 😶‍🌫️

**Comment by peterbaccaro** [comment, marked as answer]
![dental-crown](https://user-images.githubusercontent.com/74666079/230596227-01db82cf-935d-43c9-bc2e-bdf677e98a1b.png)
